HISTORY.md

```diff
@@ -11,6 +11,14 @@
 + Fix lint errors ([#786](https://github.com/ripple/ripple-lib/pull/786), [#808](https://github.com/ripple/ripple-lib/pull/808))
 + [Update ripple-keypairs dependency to 0.10.1](https://github.com/ripple/ripple-lib/pull/805)
 
+The SHA checksums for the browser version of this release can be found below.
```

Comment:
Can we use sha256 instead?

Reply:
Added in ae2ccd98873c4a1d09c2fb95dbca25e9eca28a37